pkg/products/fuse/reconciler.go

```diff
@@ -510,3 +586,29 @@ func (r *Reconciler) reconcileSubscription(ctx context.Context, serverClient k8s
 		catalogSourceReconciler,
 	)
 }
+
+//this is only a temporary function included to remove obsolete resources left from a workaround for a fuse syndesis bug, all these objects should now be generated through the ops addon
+//TODO Remove from here vvvvvvvvvvvvvvvvvv
+func (r *Reconciler) removeOldResources(ctx context.Context, serverClient k8sclient.Client, obsoleteObjects []runtime.Object) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
`obsoleteObjects` is currently a global variable that's captured in the `Reconcile` function to pass to this function, even though this function would already have access to it. That's fine and won't cause any issues, there's just a level of indirection that might not be necessary and in other cases could cause confusion, especially when the name is re-used: `obsoleteObjects` inside this function is whatever is passed into this function, and that may shadow the global variable if it's different.

Given that `obsoleteObjects` is actually only needed inside this function, my suggestion would be to declare it _inside_ this function, rather than having it be global or passed in. What do you think?